js/public/script.js

```diff
@@ -1154,17 +1154,39 @@ app.service('vCardPropertiesService', [function() {
 		email: {
 			multiple: true,
 			readableName: t('contacts', 'Email'),
-			template: 'text'
+			template: 'text',
+			defaultValue: {
+				value:[''],
+				meta:{type:['HOME']}
+			},
+			options: [
+				{id: 'HOME', name: t('contacts', 'Home')},
+				{id: 'WORK', name: t('contacts', 'Work')},
+				{id: 'OTHER', name: t('contacts', 'Other')}
+			]
 		},
 		impp: {
 			multiple: true,
 			readableName: t('contacts', 'Instant messaging'),
-			template: 'text'
+			template: 'text',
+			defaultValue: {
+				value:[''],
+				meta:{type:['HOME']}
+			},
+			options: [
+				{id: 'HOME', name: t('contacts', 'Home')},
+				{id: 'WORK', name: t('contacts', 'Work')},
+				{id: 'OTHER', name: t('contacts', 'Other')}
+			]
```

Comment:
the protocol is part of the value - https://tools.ietf.org/html/rfc6350#section-6.4.3